.github/workflows/service_test_webhdfs.yml

```diff
@@ -72,16 +72,48 @@ jobs:
       - name: Test
         shell: bash
         working-directory: core
-        run: cargo nextest run webhdfs
+        run: cargo test webhdfs
         env:
           OPENDAL_WEBHDFS_TEST: on
           OPENDAL_WEBHDFS_ROOT: /
           OPENDAL_WEBHDFS_ENDPOINT: http://127.0.0.1:9870
 
+  hdfs_with_list_batch_disabled:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Configure Webhdfs
+        # namenode will use ports: 9870 and 9000
+        # datanode will use ports: 9864
+        run: |
+          docker run -d \
+            --name namenode \
+            --network host \
+            -e CLUSTER_NAME=test \
+            -e WEBHDFS_CONF_dfs_webhdfs_enabled=true \
+            -e CORE_CONF_hadoop_http_staticuser_user=root \
+            bde2020/hadoop-namenode:2.0.0-hadoop3.1.3-java8
+
+          docker run -d \
+            --name datanode \
+            --network host \
+            -e CLUSTER_NAME=test \
+            -e WEBHDFS_CONF_dfs_webhdfs_enabled=true \
+            -e CORE_CONF_hadoop_http_staticuser_user=root \
+            bde2020/hadoop-datanode:2.0.0-hadoop3.1.3-java8
+
+          curl --retry 30 --retry-delay 1 --retry-connrefused http://localhost:9870
+
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup
+        with:
+          need-nextest: true
```

Comment:
If we are not using `nextest` should we still keep this?

btw i am curious why we are not using nextest in webhdfs

Reply:
> why we are not using nextest in webhdfs

Our non-production webhdfs setup will be overwhelmed and go down due to `nextest`s high speed...